### PR TITLE
Fix logging, pass options, fix warnings

### PR DESCRIPTION
**Pull request by justinshenk**
Fix example by adding tqdm-safe logging, nosave option, fix deprecation warnings

### CODE REVIEW

**Comment by justinshenk** [comment]
@MLRichter gentle reminder

File: delve/metrics.py [hunk 1]
```diff
@@ -61,6 +61,6 @@ def compute_diag_trace(cov: torch.Tensor) -> float:
     :param cov: the covariannce matrix as torch tensor
     :return: the trace
     """
-    eig_vals, eigen_space = cov.symeig(True)
+    eig_vals, eigen_space = torch.linalg.eigh(cov, UPLO='U')
```
Comment (MLRichter) [inline delve/metrics.py:64]: Can you explain the reasoning for this change / why is this function better (honest curiosity)?

**Review by MLRichter — APPROVED** [review]

File: delve/metrics.py (hunk 1 above)
Comment (justinshenk) [reply delve/metrics.py:64]: As noted in https://github.com/delve-team/delve/pull/38/commits/d95ff3d316a63008d63ba3d753a7ba48822ce963, there is a deprecation warning: 
/Users/justinshenk/projects/delve/delve/metrics.py:15: UserWarning: torch.symeig is deprecated in favor of torch.linalg.eigh and will be removed in a future PyTorch release.
The default behavior has changed from using the upper triangular portion of the matrix by default to using the lower triangular portion.
L, _ = torch.symeig(A, upper=upper)
should be replaced with
L = torch.linalg.eigvalsh(A, UPLO='U' if upper else 'L')
and
L, V = torch.symeig(A, eigenvectors=True)
should be replaced with
L, V = torch.linalg.eigh(A, UPLO='U' if upper else 'L') (Triggered internally at  ../aten/src/ATen/native/BatchLinearAlgebra.cpp:2500.)
  eig_vals, eigen_space = cov.symeig(True)
  
This should be tested to make sure functionality hasn't changed.

Comment (MLRichter) [reply delve/metrics.py:64]: Ah good to know.
Allright

**Pull request merged**